nmea_parser.c

```diff
@@ -92,10 +92,13 @@ void parse_NMEA_command(char* message){
 				case 'W':
 					if (*(ptr+1) == 'L') {
 						//Set Wingload/Ballast 
+                                                // we are getting total_mass / dry_mass
+                                                // we should get total_mass / reference_mass
+                                                // TODO: fix this in XCSoar
 						ptr = strtok(NULL, delimiter);
 						val = (char *) malloc(strlen(ptr));
-						strncpy(val,ptr,strlen(ptr));
-						setBallast(atof(val));
+                                                strncpy(val,ptr,strlen(ptr));
+                                                setBallast(atof(val));
```

Comment:
I think this whole patch is very inconsistent regarding spaces, tabs and indentation size. I think the original code uses tabs instead of spaces and tab size is 8.

Reply:
I'm not going to reformat it manually. All the code is pretty sloppy style.
Is there a tool to format C-code automatically? arduino does it nicely.

Reply:
Inconsistent indentation doesn't make the style less sloppy though.